portfolio/src/main/webapp/script.js

```diff
@@ -75,40 +75,48 @@ function addRandomBibleVerse() {
 }
 
 /**
- * Changes Navbar style depending on scroll position
+ * Changes Navbar style depending on scroll position.
  */
 window.addEventListener('scroll', function() {
   if (document.body.scrollTop > 80 || 
       document.documentElement.scrollTop > 80) {
     document.getElementById('navbar').style.padding = '30px 10px';
     document.getElementById('navbar').style.backgroundColor = 
         'rgba(0, 0, 0, 0.25)';
-    document.getElementById('logo').style.fontSize = '25px';
+    document.getElementById('myName').style.fontSize = '25px';
   }
   else {
     document.getElementById('navbar').style.padding = '80px 10px';
     document.getElementById('navbar').style.backgroundColor = 
         'rgba(0, 0, 0, 1)';
-    document.getElementById('logo').style.fontSize = '35px';
+    document.getElementById('myName').style.fontSize = '35px';
   }
 })
 
 /**
- * Fetches a message from the server and adds it to the DOM.
+ * Fetches all comments (submitted via the form) from the server
+ * and adds it to the DOM.
  */
-function getMessage() {
-  fetch('/data').then(response => response.json()).then((message) => {
-    const element = document.getElementById('message-container')
-    element.innerHTML = '';
-    element.appendChild(createListElement(message.string1));
-    element.appendChild(createListElement(message.string2));
-    element.appendChild(createListElement(message.string3));
+function getComment() {
+  let url = '/data?limit=' + document.getElementById("quantity").value.toString();
```

Comment:
'limit' might not be descriptive enough, perhaps 'comment-limit'?

Reply:
Ok! Shouldn't it be commentLimit, though (camel case)?
https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names

Reply:
This is true for method names and variable names, but this is a url param, and it seems like looking through the examples the preferred format is with a dash - https://github.com/OUMontiel/my-portfolio/blob/master/walkthroughs/week-3-server/examples/subtraction-game/src/main/webapp/index.html.

Glad to see you quoting the style guide though, that's awesome :D

Reply:
Done! And thanks :)